rrfs-test/validated_yamls/templates/obtype_config/atms_npp_qc_bc.yaml

```diff
@@ -194,11 +18,42 @@ cost function:
            obs options:
              Sensor_ID: atms_npp
              EndianType: little_endian
-             CoefficientPath: Data/crtm_2.4.0/
+             CoefficientPath: data/crtm_2.4.0/
              IRVISlandCoeff: IGBP
            linear obs operator:
              Absorbers: [H2O, O3]
 
+       obs bias:
+         input file:  data/obs/atms_npp.satbias.nc4
+         output file: ./out_atms_npp.satbias.nc
+         variational bc:
+           predictors:
+           - name: constant
+           - name: lapse_rate
+             order: 2
+             tlapse: &atms_npp_tlapse  data/obs/atms_npp.tlapse.txt
+           - name: lapse_rate
+             tlapse: *atms_npp_tlapse
+           - name: emissivity
+           - name: scan_angle
+             order: 4
+           - name: scan_angle
+             order: 3
+           - name: scan_angle
+             order: 2
+           - name: scan_angle
```

Comment:
- Change `lapse_rate` to `lapseRate`
- Change `emissivity` to `emissivityJacobian`
- Change `scan_angle` to `sensorScanAngle`

Reply:
> * Change `lapse_rate` to `lapseRate`
> * Change `emissivity` to `emissivityJacobian`
> * Change `scan_angle` to `sensorScanAngle`

resolved!